docusaurus/docs/how-to-guides/data-source-plugins/add-resource-handler.md

```diff
@@ -47,20 +47,27 @@ To verify the data source UID, you can enter `window.grafanaBootData.settings.da
 
 ## Add support for multiple routes
 
-To support multiple routes in your data source plugin, you can use a switch with the `req.Path`:
+To support multiple routes in your data source plugin you have a couple of options depending on requirements and needs.
```

Comment:
I guess all this is duplicated now so better to link to the new guide?

Reply:
Thanks. I'm not sure I follow. Do you mean link from add resource handler topic for apps to add resource handler topic for datasources?

Reply:
yep, the content is mostly (if not all) the same, isn't it?

Reply:
Yes true, but not sure what's wanted. The main problem/missing thing is that we cannot put topics specifically for backend plugins somewhere currently. I've been trying to vote for such a section, but seems like we haven't found a good fit for that yet. I'll check with the rest of the team, thanks

Reply:
@marefr in the meantime, what if you create a shared file and import it here? that way it looks like we have two pages with the same information but they come from the same source.